ansible/smoke.yml

```diff
@@ -150,3 +150,28 @@
       become: false
       run_once: true
       when: smoke_enable_cdn_checks | default(True)
+
+    - name: check /_tracking first redirection
+      block:
+        - name: send POST
+          uri:
+            url: https://catalog.data.gov/_tracking
+            method: POST
+            status_code: 307
+            follow_redirects: none
+      delegate_to: localhost
+      become: false
+      run_once: true
```

Comment:
Please assert the Location header is correct. You can add a `register` and then `assert` that the location is correct (pointing to admin-catalog.data.gov).
There's an example in [datagov-web](https://github.com/GSA/datagov-deploy/blob/develop/ansible/datagov-web.yml#L50-L71) (ignore the loop).

Reply:
I added the assertions: https://github.com/GSA/datagov-deploy/pull/2456/commits/c142c03cc23e671c2cd86863c45a563f83092609